src/main/resources/db/migration/V20191209120000__add_softdelete_timestamp.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE users_data
+    ADD COLUMN IF NOT EXISTS deleted TIMESTAMP WITH TIME ZONE;
```

Comment:
The column type is the TIMESPAMP - it will not be possible to set FALSE
There are existing column created/updated so "deleted" follows the same naming convetion